package.json

```diff
@@ -1,30 +1,30 @@
 {
     "name": "@morgan-stanley/message-broker",
     "version": "1.1.0",
-    "description": "Framework agnostic messagebroker for decoupled communication.",
+    "description": "Framework agnostic messageBroker for decoupled communication.",
     "main": "dist/main/index.js",
     "types": "dist/main/index.d.ts",
     "author": "Morgan Stanley",
     "license": "Apache-2.0",
     "keywords": [
-        "messagebroker",
+        "messageBroker",
         "communication",
         "event",
         "morgan stanley"
     ],
     "scripts": {
         "clean": "rimraf ./dist ./coverage ../install ../build ../packages ./reports",
-        "copy-definitions-to-dist": "copyfiles -u 1 \"./main/**/*.d.ts\" ./dist",
         "test": "vitest run",
-        "posttest": "npm run lint",
-        "lint": "eslint --ext .ts,.js .",
-        "lint-fix": "eslint --ext .ts,.js . --fix",
-        "watch-test": "vitest --watch",
-        "watch-build": "tsc --watch",
-        "build-tsc": "tsc -p tsconfig.json",
-        "build": "npm run build-tsc && npm run copy-definitions-to-dist && typedoc main/index.ts",
-        "prebuild-release": "npm run clean",
-        "build-release": "npm run build"
+        "lint": "eslint .",
+        "lint:fix": "eslint . --fix",
+        "watch:test": "vitest --watch",
+        "watch:build": "tsc --watch",
+        "build:lib": "tsc -p tsconfig.lib.json",
+        "build:specs": "tsc -p specs/tsconfig.spec.json",
+        "build": "npm run build:lib && npm run build:specs",
+        "prebuild:release": "npm run clean",
+        "build:release": "concurrently npm:build npm:test npm:lint --killOthersOn=failure",
```

Comment:
build:release script runs all tasks in parallel so one command quickly does everything we need to ensure that the build is good.